app/models/conversation.rb

```diff
@@ -38,4 +44,10 @@ def reset_unread_messages(user)
       update(second_user_unread_messages: 0)
     end
   end
+
+  private
+
+  def add_users_from_match
+    users << match.first_user << match.second_user
```

Comment:
You can pass a collection to `<<` operator

`users << [match.first_user, match.second_user]`